src/main/scala/partitioners/GroupPartitioner.scala

```diff
@@ -0,0 +1,28 @@
+package partitioners
+
+import org.apache.kafka.clients.producer.Partitioner
+import org.apache.kafka.common.Cluster
+
+class GroupPartitioner extends Partitioner {
+  private val partitionsPerGroup = 5
```

Comment:
You need to read it from the config

Reply:
I didn't understand how to do that so I put const in the meanwhile